meta-microblaze/recipes-devtools/gcc/gcc-runtime_11.3.bbappend

```diff
@@ -0,0 +1,32 @@
+# Override do_configure defined in poky/meta/recipes-devtools/gcc/gcc-runtime.inc in order to work around
```

Comment:
This commit also repairs the `-shared` issue. It's probably a better idea to backport it to kirkstone: 
https://git.yoctoproject.org/poky/commit/meta/recipes-devtools/gcc/gcc-runtime.inc?id=591f14c698f43ca4ae6431c8cd96225d1ed5dbf5

Reply:
Thanks, I triaged locally by removing this override and cherry-picking said commit on top of yocto-4.0.9 and according to my test it fixes the issue, so I sent a kirkstone backport request to oe-core mailing list: https://lists.openembedded.org/g/openembedded-core/message/181484

Reply:
Latest Kirkstone 4.0.11 now contains backported fix https://git.yoctoproject.org/poky/commit/?h=yocto-4.0.11&id=645fd39e4bf5616e887a865e69984a68defafaba so dropped override which is no longer needed from this PR